src/models/Order.js

```diff
@@ -85,10 +90,12 @@ const updateOrderStatus = async () => {
 };
 
 // Call the updateOrderStatus function every 1 minute (60 * 1000 milliseconds)
-const intervalId = setInterval(updateOrderStatus, 1 * 60 * 1000);
+
+//const intervalId = setInterval(updateOrderStatus, 1 * 60 * 1000);
+const intervalId = setInterval(updateOrderStatus, 72000000); // 2 hours
```

Comment:
Just fyi this (72,000,000) is actually **20 hours**. 2 hours would be (2 hrs \* 60 min/hr \* 60 sec/min \* 1000 millisec/sec = 7,200,000)